docs/1.0. EDID-Mapping.md

```diff
@@ -0,0 +1,220 @@
+
+  - `urn:x-nmos:cap:format:sample_depth` set to array of supported sampling bit rates as taken from EDID's CEA-861-EXTENSION-Block's Audio Data element.
+
+- MUST add a grouping tag to the created audio receiver with tag `name` and `role` set to indicate a group comprised of the video receiver and audio receiver associated with the display.
+
+- SHOULD provide a `contraint_sets` entry for each of the audio modes supported by the display as provided by the EDID as an individual `contraint_set`with parameters set as above.
+
+### EDID to NMOS IS-EDID-CM Resource
+
+IS-EDID-CM provides an API for the IS-04 EDID Device Control.  The following items apply to an NMOS device that has EDID type displays:
+
+- MUST maintain a binary representation of the EDID for each display and support GET for the endpoint `edid-cm\receiver-id\edid` where receiver-id is the IS-04 UUID for the receiver.
+
+- For each Receiver 
+- MUST populate the `edid-cm\destination` resource with the following parameters:
+
+  - `manufacturer` set to the manufacturer name taken from the display's EDID.
+
+  - `model` set to the model taken from the display's EDID.
+  
+  - `serial_number` set to the serial number taken from the display's EDID.
+  
+  - `max_image_size` Height and width set to the maximum image size taken from the display's EDID.
+  
+  - `monitor_name` taken from the EDID.
+
+### IPMX Controller
+
+The following best practices apply to an IPMX Controller:
+
+- MUST implement all User-Stories as described in the IPMX Architecture document [EDID-CM-ARCH](#edid-cm-arch-ref). 
+
+- MUST use the API specified by [EDID-CM](#edid-cm) in a way consistent with Sequence Diagrams provided the in the architecture document to realize the IPMX Architecture User Stories.
+
+- MAY be implemented as either a centralized controller or as a peer-to-peer controller.
+
+- MUST work consistently with NMOS devices that do not implement this IPMX BCP recommendations (e.g. all [IS-04](#is-04-ref), [IS-5](#is-05-ref) etc work for devices that do not provide the EDID Control)
+
+For a peer-to-peer implementation an IPMX Controller:
+
+- SHOULD provide an implementation of the [EDID-CM](#edid-cm-ref) endpoints for  retrieval by an external user.
+
+### Media Formats
+
+The following apply for choice of media formats for IPMX devices:
+
+- SHOULD use [JPEG-XS](#jpeg-xs-ref) for video compression and decompression. 
+
+- MAY use other codecs but be aware that JPEG-XS will provide best fit for IPMX.
+
+- MUST follow the specfication provided in the SDP section of the [JPEG-XS](#jpeg-xs) specification when using JPEG-XS as transport when using SDP for NMOS Connection Management [IS-05](#is-05-ref).
+
+
+### Security
+ 
+- MUST follow [BCP-003-003](#bcp-003-03-ref) for authorization of access to the EDID-CM Control and associated API.
+
+- SHOULD follow [BCP-003](#bcm-003-ref) fully for Security. 
```

Comment:
We've never spoken about these topics, I am not sure they belong in this document 🤔

Reply:
Also, they look like out-of-scope things for "EDID mapping" document.

Reply:
```suggestion
```

Reply:
We have an IPMX Architecture document - Does it make sense to move these things as recommendations or BCP for an IPMX Architecture or perhaps this  doesn't belong anywhere. Seems if we want to give guidance to Pro AV type participants then this type thing should be somewhere.

Reply:
These two section are very much "impx practices" and as other mentioned one the call they are well suited for a "TR-1001-IPMX" type document